ERROR - 2023-07-01 00:09:39 --> Error connecting to the database: mysqli_sql_exception: Unknown database 'cadastros' in C:\xampp\htdocs\twitter\rede_social-CodeIgniter\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 C:\xampp\htdocs\twitter\rede_social-CodeIgniter\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'cadastros', NULL, '', 0)
#1 C:\xampp\htdocs\twitter\rede_social-CodeIgniter\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\twitter\rede_social-CodeIgniter\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\twitter\rede_social-CodeIgniter\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\twitter\rede_social-CodeIgniter\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\twitter\rede_social-CodeIgniter\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\twitter\rede_social-CodeIgniter\app\Models\SocialMediaDB.php(40): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\twitter\rede_social-CodeIgniter\app\Controllers\SocialMedia.php(15): App\Models\SocialMediaDB->readAll()
#8 C:\xampp\htdocs\twitter\rede_social-CodeIgniter\system\CodeIgniter.php(932): App\Controllers\SocialMedia->index()
#9 C:\xampp\htdocs\twitter\rede_social-CodeIgniter\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\SocialMedia))
#10 C:\xampp\htdocs\twitter\rede_social-CodeIgniter\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\twitter\rede_social-CodeIgniter\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Unknown database 'cadastros' in C:\xampp\htdocs\twitter\rede_social-CodeIgniter\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\twitter\rede_social-CodeIgniter\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\twitter\rede_social-CodeIgniter\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\twitter\rede_social-CodeIgniter\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\twitter\rede_social-CodeIgniter\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\twitter\rede_social-CodeIgniter\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\xampp\htdocs\twitter\rede_social-CodeIgniter\app\Models\SocialMediaDB.php(40): CodeIgniter\BaseModel->findAll()
#6 C:\xampp\htdocs\twitter\rede_social-CodeIgniter\app\Controllers\SocialMedia.php(15): App\Models\SocialMediaDB->readAll()
#7 C:\xampp\htdocs\twitter\rede_social-CodeIgniter\system\CodeIgniter.php(932): App\Controllers\SocialMedia->index()
#8 C:\xampp\htdocs\twitter\rede_social-CodeIgniter\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\SocialMedia))
#9 C:\xampp\htdocs\twitter\rede_social-CodeIgniter\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\twitter\rede_social-CodeIgniter\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2023-07-01 00:09:39 --> Unable to connect to the database.
Main connection [MySQLi]: Unknown database 'cadastros'
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `usuarios`', [], false)
 3 SYSTEMPATH\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Models\SocialMediaDB.php(40): CodeIgniter\BaseModel->findAll()
 6 APPPATH\Controllers\SocialMedia.php(15): App\Models\SocialMediaDB->readAll()
 7 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\SocialMedia->index()
 8 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\SocialMedia))
 9 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-07-01 00:09:54 --> Error connecting to the database: mysqli_sql_exception: Unknown database 'cadastros' in C:\xampp\htdocs\twitter\rede_social-CodeIgniter\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 C:\xampp\htdocs\twitter\rede_social-CodeIgniter\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'cadastros', NULL, '', 0)
#1 C:\xampp\htdocs\twitter\rede_social-CodeIgniter\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\twitter\rede_social-CodeIgniter\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\twitter\rede_social-CodeIgniter\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\twitter\rede_social-CodeIgniter\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\twitter\rede_social-CodeIgniter\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\twitter\rede_social-CodeIgniter\app\Models\SocialMediaDB.php(40): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\twitter\rede_social-CodeIgniter\app\Controllers\SocialMedia.php(15): App\Models\SocialMediaDB->readAll()
#8 C:\xampp\htdocs\twitter\rede_social-CodeIgniter\system\CodeIgniter.php(932): App\Controllers\SocialMedia->index()
#9 C:\xampp\htdocs\twitter\rede_social-CodeIgniter\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\SocialMedia))
#10 C:\xampp\htdocs\twitter\rede_social-CodeIgniter\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\twitter\rede_social-CodeIgniter\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Unknown database 'cadastros' in C:\xampp\htdocs\twitter\rede_social-CodeIgniter\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\twitter\rede_social-CodeIgniter\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\twitter\rede_social-CodeIgniter\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\twitter\rede_social-CodeIgniter\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\twitter\rede_social-CodeIgniter\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\twitter\rede_social-CodeIgniter\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\xampp\htdocs\twitter\rede_social-CodeIgniter\app\Models\SocialMediaDB.php(40): CodeIgniter\BaseModel->findAll()
#6 C:\xampp\htdocs\twitter\rede_social-CodeIgniter\app\Controllers\SocialMedia.php(15): App\Models\SocialMediaDB->readAll()
#7 C:\xampp\htdocs\twitter\rede_social-CodeIgniter\system\CodeIgniter.php(932): App\Controllers\SocialMedia->index()
#8 C:\xampp\htdocs\twitter\rede_social-CodeIgniter\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\SocialMedia))
#9 C:\xampp\htdocs\twitter\rede_social-CodeIgniter\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\twitter\rede_social-CodeIgniter\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2023-07-01 00:09:54 --> Unable to connect to the database.
Main connection [MySQLi]: Unknown database 'cadastros'
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `usuarios`', [], false)
 3 SYSTEMPATH\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Models\SocialMediaDB.php(40): CodeIgniter\BaseModel->findAll()
 6 APPPATH\Controllers\SocialMedia.php(15): App\Models\SocialMediaDB->readAll()
 7 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\SocialMedia->index()
 8 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\SocialMedia))
 9 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-07-01 00:10:18 --> Error connecting to the database: mysqli_sql_exception: Unknown database 'cadastros' in C:\xampp\htdocs\twitter\rede_social-CodeIgniter\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 C:\xampp\htdocs\twitter\rede_social-CodeIgniter\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'cadastros', NULL, '', 0)
#1 C:\xampp\htdocs\twitter\rede_social-CodeIgniter\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\twitter\rede_social-CodeIgniter\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\twitter\rede_social-CodeIgniter\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\twitter\rede_social-CodeIgniter\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\twitter\rede_social-CodeIgniter\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\twitter\rede_social-CodeIgniter\app\Models\SocialMediaDB.php(40): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\twitter\rede_social-CodeIgniter\app\Controllers\SocialMedia.php(15): App\Models\SocialMediaDB->readAll()
#8 C:\xampp\htdocs\twitter\rede_social-CodeIgniter\system\CodeIgniter.php(932): App\Controllers\SocialMedia->index()
#9 C:\xampp\htdocs\twitter\rede_social-CodeIgniter\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\SocialMedia))
#10 C:\xampp\htdocs\twitter\rede_social-CodeIgniter\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\twitter\rede_social-CodeIgniter\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Unknown database 'cadastros' in C:\xampp\htdocs\twitter\rede_social-CodeIgniter\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\twitter\rede_social-CodeIgniter\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\twitter\rede_social-CodeIgniter\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\twitter\rede_social-CodeIgniter\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\twitter\rede_social-CodeIgniter\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\twitter\rede_social-CodeIgniter\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\xampp\htdocs\twitter\rede_social-CodeIgniter\app\Models\SocialMediaDB.php(40): CodeIgniter\BaseModel->findAll()
#6 C:\xampp\htdocs\twitter\rede_social-CodeIgniter\app\Controllers\SocialMedia.php(15): App\Models\SocialMediaDB->readAll()
#7 C:\xampp\htdocs\twitter\rede_social-CodeIgniter\system\CodeIgniter.php(932): App\Controllers\SocialMedia->index()
#8 C:\xampp\htdocs\twitter\rede_social-CodeIgniter\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\SocialMedia))
#9 C:\xampp\htdocs\twitter\rede_social-CodeIgniter\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\twitter\rede_social-CodeIgniter\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2023-07-01 00:10:18 --> Unable to connect to the database.
Main connection [MySQLi]: Unknown database 'cadastros'
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `usuarios`', [], false)
 3 SYSTEMPATH\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Models\SocialMediaDB.php(40): CodeIgniter\BaseModel->findAll()
 6 APPPATH\Controllers\SocialMedia.php(15): App\Models\SocialMediaDB->readAll()
 7 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\SocialMedia->index()
 8 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\SocialMedia))
 9 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
